### PR TITLE
Add packages to compile on the bubble itself

### PR DESCRIPTION
**Pull request by bheuvel**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Recipes controlled by attributes
 include_recipe 'bubble::packages' if node['bubble']['packages']
+group node['bubble']['group_name']
```
Comment (borisroman) [inline recipes/default.rb:3]: @bheuvel Why is this needed?

Comment (bheuvel) [inline recipes/default.rb:3]: @borisroman This group is used in several recipes as owner, but nowhere the group is created. Except (default) when the users recipe is included. The example databag "happens" to have the same groupname filled in with the user, hence created.

When running this on a "vanilla" centos box, and not using the users recipe, it will fail.
